### PR TITLE
webUI small fixes and improvements

### LINKED ISSUE #976: WebUI: no elements in sequence with empty token list
## Problem Definition

When opening the token list in the WebUI after a fresh deploy of the `Registry` contract the following error is shown in red in the notification area:

`Raiden Error
no elements in sequence`

Steps to reproduce:
1. Start raiden with the following (empty) registry contract: 
  `raiden --registry-contract-address 0xf5882a8d69d16c6dd8e82292e785807beb122f5a`
1. Visit Token list on WebUI

## Solution

Don't expect the token list to always have entries.

## Tasklist

- [ ] Don't expect the token list to always have entries.

### PR DESCRIPTION
**Pull request by andrevmatos**
- Fix #976 by supporting empty token list.
- Increase timeout to 10min, as with #967 , it should support long running transactions (like deposits).
- Disable invalid actions in channels menus depending on channel state.
- Updated webUI dependencies to latest stable versions.

### CODE REVIEW

File: raiden/ui/web/src/app/services/raiden.service.ts [hunk 1]
```diff
@@ -48,7 +47,7 @@ export class RaidenService {
     }
 
     get identifier(): number {
-        return this._identifier++;
```
Comment (LefterisJP) [inline raiden/ui/web/src/app/services/raiden.service.ts]: Where is this identifier going? Is it the identifier for the transfers?

Comment (andrevmatos) [reply raiden/ui/web/src/app/services/raiden.service.ts]: Yes, it's used also for transfers. As I stated on #935 , webUI is going to use timestamp-based + random, for generating identifiers (as I said, a common pattern in web development).

Comment (LefterisJP) [reply raiden/ui/web/src/app/services/raiden.service.ts]: But is that really needed? If you don't provide an identifier with a transfer, a random one is provided in the backend as can be seen [here](https://github.com/raiden-network/raiden/blob/e5c3ae927b170de06a3a9b1d0c7b5819f2b800b7/raiden/raiden_service.py#L98)

Comment (andrevmatos) [reply raiden/ui/web/src/app/services/raiden.service.ts]: Yes, I know, but from that discussion, we went that the client is the responsible for providing an identifier, if it wants to keep track of the request. This is the base for the whole "idempotent" requests, even if it's not really idempotent yet. IF the client doesn't provide one, we'll assign a random, but a well implemented client should always keep track and provide it itself. It's even more important if we plan to support things like websockets in the future.

Comment (LefterisJP) [reply raiden/ui/web/src/app/services/raiden.service.ts]: Okay I see, so you generate the random identifier your way and also store it in the webui to keep track of the request?

Comment (andrevmatos) [reply raiden/ui/web/src/app/services/raiden.service.ts]: For now, it's stored only in the _flying_ request, and discarded when it completes (and a new one is generated for each request), but, in the near future, I plan to have a central store of it, allowing, for example, to cancel long-running requests on user's command.

**Pull request merged**